src/docs/content/docs/usage/sidecar_mode.md

```diff
@@ -0,0 +1,49 @@
+It is a pattern that is often used in Kubernetes, where the main application and the sidecar application are deployed as separate containers in the same pod.
+
+In Sidecar Mode, each Cassandra node process is deployed alongside a Reaper process; Cassandra is the parent application and Reaper is the sidecar.
+
+## Advantages
+
+ * *Security*: Each Reaper process only needs local JMX access. No need to configure remote JMX access and JMX authentication.
+ * *Fault Tolerance*: Since there is one Reaper process per node, the Reaper services will be as resilient as the parent Cassandra cluster.
+ * *Kubernetes Friendly*: Sidecar mode is very easy to setup if your Cassandra Cluster is deployed using a container orchestration system, such as Kubernetes.
+
+
+## Guidance
+
+ * *Deploy one Reaper cluster per Cassandra cluster*: Sidecar mode has been designed to allow you to easily deploy a separate, highly available Reaper service for each of your Cassandra clusters.
+   You can use a Sidecar Mode Reaper service to manage multiple clusters, but this is not the use-case for which it has been designed and tested.
+ * *Deploy Reaper in a sidecar container alongside Cassandra*: If you are using Kubernetes to deploy Cassandra and Reaper, put the Reaper process in the same Pod as Cassandra, in a sidecar container. See [Pods that run multiple containers that need to work together](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/)
+ * *Use Spreaper and the REST API for Reaper administration (use the WebUI for reporting)*: If you have automated the deployment of Reeaper in Sidecar Mode alongside a Cassandra Cluster (described above), you can also [Preconfigure that cluster](https://github.com/thelastpickle/cassandra-reaper/pull/425) by writing a script that interacts with the Reaper REST API (or using spreaper) to automatically configure the repair settings for that cluster.
+ * *Use or write an operator*: For example, you could write a [Kubernetes Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (e.g. a `Cassandra` resource) which contains configuration fields for the Cassandra database and fields with which you can define the Reaper repair settings.
+   Then write an accompanying [Kubernetes Operator](https://coreos.com/operators/) which watches for changes to those resources and reconciles the declared (desired) configuration with the actual state of the Cassandra cluster and its Reaper service.
+
+## Caveats
+
+ * *Cassandra Backend Only*: When operating in this mode, Reaper must be configured to use the [Cassandra Backend](../../backends/cassandra).
+ * *One Reaper cluster per Cassandra Cluster*: Sidecar Mode is designed to be used in situations where you deploy a separate Reaper cluster for every Cassandra cluster.
+   If you prefer to deploy a single Reaper cluster to manage multiple Cassandra clusters then Sidecar Mode may not be suitable for you.
```

Comment:
I have to disagree here, we're still aiming at achieving federation of Reaper instances so that we can centralize repair management across the fleet.